### PR TITLE
docs(scc-samples): update comments to include new parent types

### PR DESCRIPTION
**Pull request by Sita04**
# Security Command Center Samples

Contains only comment updates to allow new parent types.

### CODE REVIEW

File: google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/AssetSnippets.java [hunk 1]
```diff
@@ -46,8 +50,12 @@ private AssetSnippets() {}
   // [START securitycenter_list_all_assets]
   static ImmutableList<ListAssetsResult> listAssets(OrganizationName organizationName) {
     try (SecurityCenterClient client = SecurityCenterClient.create()) {
-      // Start setting up a request for to search for all assets in an organization.
-      // OrganizationName organizationName = OrganizationName.of(/*organizationId=*/"123234324");
+      // Start setting up a request for to search for all assets in an organization/ project/ folder.
```
Comment (bpj146) [inline google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/AssetSnippets.java:53]: delete extra "for"

File: google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/AssetSnippets.java
Comment (bpj146) [inline google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/AssetSnippets.java:1]: Just a heads up that the SCC Assets API is deprecated and will be replaced by integrating Cloud Asset Inventory's API in SCC. The CAI replacement is expected to GA at the beginning of August. Existing users have a year to move off of the SCC API. SCC's Assets API is disabled for all new users who activate SCC after last week's deprecation notice. New users can only use CAI.

File: google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/FindingSnippets.java [hunk 2]
```diff
@@ -226,7 +230,14 @@ static Finding setFindingState(FindingName findingName) {
   // [START securitycenter_list_all_findings]
   static ImmutableList<ListFindingsResult> listAllFindings(OrganizationName organizationName) {
     try (SecurityCenterClient client = SecurityCenterClient.create()) {
-      // OrganizationName organizationName = OrganizationName.of(/*organizationId=*/"123234324");
+      // SourceName must be in one of the following formats:
```
Comment (bpj146) [inline google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/FindingSnippets.java:233]: Should this really say "SourceName"? The list that follows seems to show only the format of the org, folder, or project specification.

File: google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/SecurityMarkSnippets.java [hunk 3]
```diff
@@ -44,7 +45,11 @@ private SecurityMarkSnippets() {}
   // [START securitycenter_add_security_marks]
   static SecurityMarks addToAsset(String assetName) {
     try (SecurityCenterClient client = SecurityCenterClient.create()) {
-      // String assetName = "organizations/123123342/assets/12312321";
+      // assetName: must be in one of the following formats:
```
Comment (bpj146) [inline google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/SecurityMarkSnippets.java:48]: "Specify the value of assetName in one of the following formats:"?

Comment (Sita04) [reply google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/SecurityMarkSnippets.java]: Done. Thanks

File: google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/FindingSnippets.java (hunk 2 above)
Comment (Sita04) [reply google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/FindingSnippets.java]: Done. Thanks

File: google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/SecurityMarkSnippets.java [hunk 4]
```diff
@@ -228,8 +244,14 @@ static ImmutableList<ListAssetsResult> listAssetsWithQueryMarks(
   static ImmutableList<ListFindingsResult> listFindingsWithQueryMarks(SourceName sourceName) {
     try (SecurityCenterClient client = SecurityCenterClient.create()) {
       // Start setting up a request for to list all findings filtered by a specific security mark.
-      // SourceName sourceName = SourceName.of(/*organization=*/"123234324",/*source=*/
-      // "423432321");
+      // SourceName must be in one of the following formats:
```
Comment (bpj146) [inline google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/SecurityMarkSnippets.java:247]: Same issue as before, where it seems like the list is showing the format of the parent specification, not the SourceName specification.

File: google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/SourceSnippets.java [hunk 5]
```diff
@@ -32,9 +32,13 @@
 import com.google.protobuf.FieldMask;
 import java.io.IOException;
 
-/** Snippets for how to work with Sources in Cloud Security Command Center. */
+/**
+ * Snippets for how to work with Sources in Cloud Security Command Center.
```
Comment (bpj146) [inline google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/SourceSnippets.java:36]: "Cloud Security Command Center" is old. Should just be "Security Command Center"

File: google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/SecurityMarkSnippets.java (hunk 4 above)
Comment (Sita04) [reply google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/SecurityMarkSnippets.java:247]: Corrected it. Thank for the suggestion.

File: google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/SourceSnippets.java (hunk 5 above)
Comment (Sita04) [reply google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/SourceSnippets.java:36]: Updated it across files. Thanks.

File: google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/AssetSnippets.java [hunk 6]
```diff
@@ -46,8 +50,12 @@ private AssetSnippets() {}
   // [START securitycenter_list_all_assets]
   static ImmutableList<ListAssetsResult> listAssets(OrganizationName organizationName) {
     try (SecurityCenterClient client = SecurityCenterClient.create()) {
-      // Start setting up a request for to search for all assets in an organization.
-      // OrganizationName organizationName = OrganizationName.of(/*organizationId=*/"123234324");
+      // Start setting up a request to search for all assets in an organization/ project/ folder.
```
Comment (bpj146) [inline google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/AssetSnippets.java:53]: Global comment: Can we replace the slashes with commas?: "...in an organization, project, or folder."

This text appears in at least 7 places in the file.

File: google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/AssetSnippets.java [hunk 7]
```diff
@@ -75,8 +83,12 @@ static ImmutableList<ListAssetsResult> listAssets(OrganizationName organizationN
   // [START securitycenter_list_assets_with_filter]
   static ImmutableList<ListAssetsResult> listAssetsWithFilter(OrganizationName organizationName) {
     try (SecurityCenterClient client = SecurityCenterClient.create()) {
-      // Start setting up a request for to search for all assets in an organization.
-      // OrganizationName organizationName = OrganizationName.of(/*organizationId=*/"123234324");
+      // Start setting up a request to search for all assets in an organization/ project/ folder.
```
Comment (bpj146) [inline google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/AssetSnippets.java:86]: replace slashes with commas and add "or"?: "...organization, project, or folder."

File: google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/AssetSnippets.java [hunk 8]
```diff
@@ -109,10 +121,13 @@ static ImmutableList<ListAssetsResult> listAssetsWithFilter(OrganizationName org
   static ImmutableList<ListAssetsResult> listAssetsAsOfYesterday(
       OrganizationName organizationName, Instant asOf) {
     try (SecurityCenterClient client = SecurityCenterClient.create()) {
-      // Start setting up a request for to search for all assets in an organization.
-      // OrganizationName organizationName = OrganizationName.of(/*organizationId=*/"123234324");
-
-      // Initialize the builder with the organization and filter
+      // Start setting up a request to search for all assets in an organization/ project/ folder.
```
Comment (bpj146) [inline google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/AssetSnippets.java:124]: "/" => "," + "or"

File: google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/AssetSnippets.java [hunk 9]
```diff
@@ -223,9 +246,10 @@ static ImmutableList<GroupResult> groupAssets(OrganizationName organizationName)
   // [START securitycenter_group_all_assets_with_filter]
   static ImmutableList<GroupResult> groupAssetsWithFilter(OrganizationName organizationName) {
     try (SecurityCenterClient client = SecurityCenterClient.create()) {
-      // Start setting up a request for to filter all assets by type and group them by project in an
+      // Start setting up a request to filter all assets by type and group them by project in an
       // organization.
-      // OrganizationName organizationName = OrganizationName.of("123234324");
+      // You can also use a project/ folder as a parent resource and filter assets in them
```
Comment (bpj146) [inline google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/AssetSnippets.java:251]: change "project/ folder" with "project or folder"?

File: google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/FindingSnippets.java [hunk 10]
```diff
@@ -353,12 +370,18 @@ static TestIamPermissionsResponse testIamPermissions(SourceName sourceName) {
    * Group all findings under an organization across all sources by their specified properties (e.g.
    * category).
    *
-   * @param organizationName The organizatoin to group all findings for.
+   * @param organizationName The organization to group all findings for.
```
Comment (bpj146) [inline google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/FindingSnippets.java:373]: Should there be a colon after organizationName?: "organizationName: The organization to..."

File: google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/SourceSnippets.java [hunk 11]
```diff
@@ -74,8 +78,11 @@ static Source createSource(OrganizationName organizationName) {
   // [START securitycenter_list_sources]
   static ImmutableList<Source> listSources(OrganizationName organizationName) {
     try (SecurityCenterClient client = SecurityCenterClient.create()) {
-      // Start setting up a request to list sources in an organization.
-      // OrganizationName organizationName = OrganizationName.of(/*organizationId=*/"123234324");
+      // Start setting up a request to list sources in an organization/ project/ folder.
```
Comment (bpj146) [inline google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/SourceSnippets.java:81]: Change to "organization, project, or folder."?

**Review by suztomo — APPROVED** [review]
Shall we start thinking about moving to java-docs-samples?

**Review by iamannalynleynes1206 — APPROVED** [review]

**Pull request merged**